app/App.js

```diff
@@ -3,9 +3,10 @@ import { connect } from 'react-redux'
 import Tracker from './Tracker'
 
 class App extends Component {
-  static propTypes = {
```

Comment:
This is really experimental syntax? I might have to dispute your claim that it shouldn't be used, I think this syntax is in very wide use in the ES6 community.